FrontEnd/src/components/SearchPage/Search.js

```diff
@@ -0,0 +1,109 @@
+import './search_page.css';
```

Comment:
I would put it at the bottom

Reply:
Search page it's another task, but thanks
Fixed